src/Settings/FlyView.SettingsGroup.json

```diff
@@ -66,6 +66,12 @@
     "units":            "m",
     "default":     1000,
     "min":              1
+},
+{
+    "name":             "updateHomePosition",
+    "shortDesc": "Update GCS' homeposition coordinates in case of change on Autopilot",
```

Comment:
Should be "home position" not homeposition.

The description is a bit confusing. Maybe it should be "Send updated GCS' home position to autopilot in case of change of home position."

Reply:
see commit https://github.com/mavlink/qgroundcontrol/pull/10085/commits/776ada0bef6f91345b4e5d7368137c9e94be5a54